.github/workflows/partial-tests.yml

```diff
@@ -9,14 +9,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: ricardochaves/python-lint@v1.4.0
+      - uses: jpetrucciani/mypy-check@master
```

Comment:
This action uses mypy 1.9.0. I like it :D

Reply:
Yeah, I chose this one because it updates quite often. Now to fix the erros so the linter passes